app/main.py

```diff
@@ -1 +1,12 @@
 # write your code here
+def copy_file(cmd: str) -> None:
+    cmd_list = cmd.split()
+
+    if len(cmd_list) == 3 and cmd_list[0] == "cp":
+        _, existing_file, new_file = cmd_list
+
+        if existing_file != new_file:
+
+            with open(existing_file, "r") as file_in:
+                with open(new_file, "w") as file_out:
```

Comment:
Allocate one context manager initializing several arguments in it as it was done before, avoid nesting of context managers where possible.